mesonbuild/dependencies/misc.py

```diff
@@ -50,6 +50,27 @@ def netcdf_factory(env: 'Environment',
 packages['netcdf'] = netcdf_factory
 
 
+class AtomicBuiltinDependency(BuiltinDependency):
+    def __init__(self, name: str, env: 'Environment', kwargs: T.Dict[str, T.Any]):
```

Comment:
Nit: the codebase now depends on python 3.7 and as such, uses `from __future__ import annotations` everywhere. That means we don't need to quote annotations to defer them anymore. We should use the new style, i.e. avoid the quotes, when adding brand-new code